engines/bladerunner/script/scene/ug15.cpp

```diff
@@ -40,7 +40,7 @@ void SceneScriptUG15::InitializeScene() {
 	} else {
 		Setup_Scene_Information(-238.0f, 48.07f,  222.0f, 180);
 		if (Game_Flag_Query(kFlagUG15RatShot)
-		 && Random_Query(1, 10) == 10
+		 && (Random_Query(1, 10) == 10)
```

Comment:
this changes nothing except of improved readability, commit message is a bit misleading